src/main/java/io/logz/jmx2graphite/Jmx2GraphiteConfiguration.java

```diff
@@ -2,52 +2,82 @@
 
 import com.typesafe.config.Config;
 
+import java.net.InetAddress;
 import java.net.MalformedURLException;
 import java.net.URL;
+import java.net.UnknownHostException;
 import java.util.concurrent.TimeUnit;
 
 /**
  * @author amesika
  */
 public class Jmx2GraphiteConfiguration {
     /* Jolokia full URL */
-    public String jolokiaUrl;
+    private String jolokiaUrl;
 
-    public String graphiteHostname;
-
-    public int graphitePort;
+    // Graphite server related params
```

Comment:
you can create inner class GraphiteServer or just Graphite and embed those two variables there and delete the comment above.